tools/run-clang-tidy-in-ci.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -ex
+
+if [[ ! -d build ]]; then
```

Comment:
...err... when is this conditional ever false?

Reply:
When you run the script many times locally